### PR TITLE
[zh] Localize context-propagation.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh/docs/concepts/context-propagation.md

### CODE REVIEW

File: content/zh/docs/concepts/context-propagation.md
Comment (my-git9) [reply content/zh/docs/concepts/context-propagation.md:13]: I think it would be easier to review if the original text is annotated here. Otherwise, the reviewer needs to open the original text to compare, which makes the review work more cumbersome. What do you think?

**Review by windsonsea — APPROVED** [review]
The [preview](https://deploy-preview-4536--opentelemetry.netlify.app/zh/docs/concepts/context-propagation/) looks good.

File: content/zh/docs/concepts/context-propagation.md
Comment (windsonsea) [reply content/zh/docs/concepts/context-propagation.md:13]: Fine, thanks, it can provide some more conveniences.

File: content/zh/docs/concepts/context-propagation.md
Comment (reyang) [inline content/zh/docs/concepts/context-propagation.md:70]:
```suggestion
**传播**是上下文在服务和进程之间移动的机制。
```

File: content/zh/docs/concepts/context-propagation.md
Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:13]: I second @windsonsea comment, this should be removed in the spirit of keeping things [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Otherwise, how do you know that the comment is up-to-date? You'll need to open the original file to double check anyways.

Please remove all embedded HTML comments.

What would be more useful would be to include, in the front matter, a **permalink** to the specific version (including the commit hash) of the `en` page that was used to create this translation, but we can discuss this separately. /cc @open-telemetry/docs-approvers

**Review by chalin — CHANGES_REQUESTED** [review]
Hi @my-git9: thanks for the translation. See the inline comment, and also please rebase from `main`.

File: content/zh/docs/concepts/context-propagation.md
Comment (my-git9) [reply content/zh/docs/concepts/context-propagation.md:13]: I still stick to my point of view for the following reasons:
1. During the first synchronization, we can easily compare with the original text one-to-one, but during the second or subsequent synchronization, it is difficult for both the submitter and the reviewer to find the differences between the translation and the original text without comments. Adding comments is not for the comparison of the first translation, but for the convenience of comparison during subsequent synchronization. It allows us to more easily locate the differences between the translation and the original text. Otherwise, even if we know that the two articles are different, it is difficult to find the differences if the articles are very long.
2. If we continue to keep the translation consistent with the original text, we should make the subsequent work simpler so that more people can participate in it (I think adding comments can make the subsequent synchronization work simpler)
3. I personally think that the localization of k8s documents is a better example: https://github.com/kubernetes/website/blob/main/content/zh-cn/docs/contribute/localization.md?plain=1

Comment (windsonsea) [reply content/zh/docs/concepts/context-propagation.md:13]: IMO, it appears to be an advanced option in future, but shall not be mandatory currently.
K8s only applies this policy to the zh-cn localization and not to other languages.

Can we simply follow standard localization practices at the start?

Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:13]: Thank you for providing more context, and the reference to the K8s example.

FYI, I've started two threads of discussion in the hopes of figuring out what "best practices" there might already be in use at the CNCF or elsewhere:

- https://github.com/open-telemetry/opentelemetry.io/discussions/4302#discussioncomment-9616612
- https://cloud-native.slack.com/archives/CUJ6W5TLM/p1717149935431579

Let me respond more to your suggestion to embed `en` page segments as HTML comments in `zh` pages in the thread of https://github.com/open-telemetry/opentelemetry.io/discussions/4302#discussioncomment-9616612.

Comment (svrnm) [reply content/zh/docs/concepts/context-propagation.md:13]: is this blocking for merging this PR?

Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:13]: Yes, I'd like to propose that you give me a couple of days to explore alternatives and make a concrete proposal for use in this repo. I've created

- #4582

to track this issue. Here's an excerpt:

> Our support for translations is nascent. We have an opportunity to get it right, or at least good enough, from the start.
> 
> Let me put something together to show you what I have in mind. In the meantime, I'd suggest delaying the merge of other zh page translations.

@svrnm, WDYT?

Comment (svrnm) [reply content/zh/docs/concepts/context-propagation.md:13]: Thanks @chalin 

@my-git9 we discussed this during yesterdays SIG comms meeting and for now we do not want to have the english version inlined. As @chalin stated our support for translation is young and experimental and we do not want to make such a decision right now, as removing the blocks of english translations after the fact is harder than adding them **if** we decide to apply this technique.

See https://github.com/open-telemetry/opentelemetry.io/issues/4582 for more details

File: content/zh/docs/concepts/context-propagation.md
Comment (reyang) [inline content/zh/docs/concepts/context-propagation.md:9]: It seems we have different translations for the same word, would be good to keep consistency. https://github.com/open-telemetry/opentelemetry.io/blob/43fea35b57456a18a0ef4b78072819f76a1c4fdd/content/zh/docs/what-is-opentelemetry.md?plain=1#L9

Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:9]: Is this change blocking @reyang?

**Review by chalin — APPROVED** [review]
LGTM if @reyang is happy 😄

File: content/zh/docs/concepts/context-propagation.md
Comment (reyang) [reply content/zh/docs/concepts/context-propagation.md:9]: Not blocking, we can cover it in separate PRs.

Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:9]: Ok TY, merging!

Comment (chalin) [reply content/zh/docs/concepts/context-propagation.md:9]: Issue:

- #4603

**Pull request merged**
